src/steps/base.py

```diff
@@ -56,6 +56,7 @@ def _prep_cache(self, cache_dirpath):
         self.cache_filepath_step_transformer = os.path.join(self.cache_dirpath_transformers, self.name)
         self.save_filepath_step_output = os.path.join(self.save_dirpath_outputs, '{}'.format(self.name))
         self.save_filepath_step_tmp = os.path.join(self.save_dirpath_tmp, '{}'.format(self.name))
+        self._cached_output = None
```

Comment:
@taraspiotr so now we don't have this load/save cache option available at all. It is ok for this project but for other projects (general use) I'd rather have that available so when you make a PR to steppy make sure to deal with both options. That suggest it is probably a good spot to deal with both here not to have vastly different codebase.

Reply:
@jakubczakon we still have an option to save output on disk, but yes, cache in memory is implemented in place of cache on disk, with the same interface. Right know in steppy cache and save are pretty much the same with the only difference being that cache is saved in temp directory. I think it is worth rethinking if caching on disk alongside saving on disk is not a bit redundant.

Reply:
@taraspiotr we've discussed and figured that the difference between cache and save is that cache is during one run (cleaned after) saved is done between runs (no cleaning). If you do decide to save you need to load it on purpuse as well. You can also load those pickle objects somewhere else (notebook) to investigate. So I think we potentially all of the cases dealt with (cache on disk, cache in memory, save on disk).

I will merge it here but PR for steppy would need to be a bit more general.

Reply:
@jakubczakon ok, sure